udmis/src/main/java/com/google/bos/udmi/monitoring/LogTailBase.java

```diff
@@ -0,0 +1,10 @@
+package com.google.bos.udmi.monitoring;
+
+import com.google.bos.udmi.service.pod.ContainerBase;
+
+/**
+ * Extend ContainerBase with UDMI Monitoring specific functions.
+ */
+public class LogTailBase extends ContainerBase {
```

Comment:
NB: I don't understand the purpose of this class, but it's not hurting anything.